scss/reprivata-theme/custom.scss

```diff
@@ -185,3 +195,56 @@ $display-font-family: $work-sans;
 $lead-font-size: $font-size-base * 1.125;
 $lead-font-weight: 400;
 $small-font-size: 0.889rem;
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
+
+// scss-docs-start input-btn-variables
+
+$input-btn-padding-y: 8px;
+$input-btn-padding-x: 16px;
+$input-btn-font-family: $open-sans;
+$input-btn-font-size: $small-font-size;
+$input-btn-line-height: 14px;
```

Comment:
Line heights need to be cohesive within an application, or you end up with all kinds of mismatch issues involving height and alignment. 

Bootstrap specifies a universal line height for this very reason:
```
$input-btn-line-height:       $line-height-base !default;
```